src/cf.data.pre

```diff
@@ -2871,6 +2871,11 @@ DOC_START
 	Processing proceeds in the order specified, and stops at first fully
 	matching line.
 
+	The ipaddr may be one of the following labels (instead of a literal IP address):
+	    transparent: Sets the outgoing IP address to the same IP as the client's connection to the proxy.
```

Comment:
"Client's connection to the proxy" has two IP addresses (source and destination), not one. Please rephrase to clearly indicate which of those two IP addresses Squid will try to use. I know that the _example_ below can be interpreted to answer that question, but we should not rely on the example for such a key determination IMO.

Reply:
I've added a slight clarification here in my latest commit, but it's worth noting the concise example that comes just below that text should clear up any misconceptions that users might have.